apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/CoverageTransformerV2.java

```diff
@@ -93,21 +89,6 @@ private static Coverage transformPartA(MetricRegistry metricRegistry, Beneficiar
         .getMedicareCoverageStartDate()
         .ifPresent(value -> TransformerUtilsV2.setPeriodStart(coverage.getPeriod(), value));
 
-    // deh start
```

Comment:
This makes sense to remove.  The coverage.contain doesn't even show up in the golden bene data.  Plus, coverage.contract isn't required by CARIN or HL7 US-CORE and this is hard coded anyway.